README.md

```diff
@@ -11,6 +11,23 @@
 ### Purpose
 Lightweight UITests framework based on XCTest to help you write and maintain UITests quickly and easily with well organised code structure.
 
+### Use Swift Package Manager as a dependancy manager 
+
+The Swift Package Manager is a tool for automating the distribution of Swift code and is integrated into the swift compiler. To add a package dependency to your Xcode project, select File > Swift Packages > Add Package Dependency and enter https://github.com/treatwell/twuitests.git
+
+or add dependency to your `Package.swift`
+
+```swift
+import PackageDescription
+
+let package = Package(
+    name: "MySPMLibrary",
+    dependencies: [
+        .package(url: "https://github.com/treatwell/twuitests.git", .upToNextMajor(from: "0.2.0"))
```

Comment:
You can remove `.upToNextMajor(` and use just `from: "0.2.0"`. Means the same.

Reply:
👍

Reply:
Is the change from 2 to 20 intentional? @treatwell-marius

Reply:
Yes 😄 I am going to bump version to `0.20.0`, current one is `0.17`